### DISCUSSION TITLE
App crashes without any warning in Android : React-Native: 0.73.6, RNGH: 2.16.0

### DISCUSSION THREAD

**Discussion by Mohamed-Ziyad** [software-mansion/react-native-gesture-handler#2842] [Q&A]
App.js
<img width="472" alt="Screenshot 1445-09-22 at 14 13 38" src="https://github.com/software-mansion/react-native-gesture-handler/assets/48976332/afa138d6-4620-4487-aabb-dbc62d164ece">

Component with gesture - this component will be render conditionally 
<img width="574" alt="Screenshot 1445-09-22 at 14 14 50" src="https://github.com/software-mansion/react-native-gesture-handler/assets/48976332/c91658c4-de06-4754-95e2-8c5a80cc95bb">

Menu2 has the gesture 
<img width="759" alt="Screenshot 1445-09-22 at 14 16 55" src="https://github.com/software-mansion/react-native-gesture-handler/assets/48976332/66986aab-f065-4940-a3f7-9bab233c64ce">

crash description from android logcat.
![Screenshot 1445-09-22 at 14 18 15](https://github.com/software-mansion/react-native-gesture-handler/assets/48976332/6eaadcc8-896d-4785-9674-817bdc2223ae)

How to fix it? any patches available?

**Comment by hparyani49** [comment]
Hi, were you able to fix this?

**Reply by Mohamed-Ziyad** [reply]
yes

**Reply by hparyani49** [reply]
Can you share what was the issue and the fix you did?

**Reply by Mohamed-Ziyad** [reply]
I installed the compatible gesture version. but I forgot whats the version is because now I'm using RN 0.76.6 and RNGH 2.9.0.
